README.md

```diff
@@ -61,6 +61,13 @@ Invoke "whoami.exe" to demonstrate running a command as another user:
   whoami.run_command
 ```
 
+Invoke "whoami.exe" with elevated privileges:
+
+```ruby
+  whoami = Mixlib::ShellOut.new("whoami.exe", :user => "username", :domain => "DOMAIN", :password => "password", :elevated => "true")
```

Comment:
`elavated` accepts only boolean. So it should be `:elevated => true` instead of `:elevated => "true"`.